suite-native/atoms/src/Switch.tsx

```diff
@@ -9,46 +9,54 @@ import Animated, {
 } from 'react-native-reanimated';
 
 import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
-import { Color } from '@trezor/theme';
+import { palette } from '@trezor/theme';
 
 import { ACCESSIBILITY_FONTSIZE_MULTIPLIER } from './Text';
 
 type SwitchProps = {
     isChecked: boolean;
     onChange: (value: boolean) => void;
     isDisabled?: boolean; // Functionality of disabled works but styles are not implemented yet (waiting for design)
-    colorChecked?: Color;
-    colorUnchecked?: Color;
 };
 
-const SWITCH_CONTAINER_WIDTH = 44 * ACCESSIBILITY_FONTSIZE_MULTIPLIER;
-const SWITCH_CONTAINER_HEIGHT = 24 * ACCESSIBILITY_FONTSIZE_MULTIPLIER;
+const SWITCH_CONTAINER_BORDER_WIDTH = 1 * ACCESSIBILITY_FONTSIZE_MULTIPLIER;
+const SWITCH_CONTAINER_WIDTH =
+    44 + 2 * SWITCH_CONTAINER_BORDER_WIDTH * ACCESSIBILITY_FONTSIZE_MULTIPLIER;
```

Comment:
Should not it be?
```suggestion
    44 * ACCESSIBILITY_FONTSIZE_MULTIPLIER + 2 * SWITCH_CONTAINER_BORDER_WIDTH * ACCESSIBILITY_FONTSIZE_MULTIPLIER;
```

Same for `SWITCH_CONTAINER_HEIGHT`.

Was is tested with upscaled font?

Reply:
good catch

Reply:
063e8ea369

Reply:
![Uploading Screenshot_1723631365.png…]()

Reply:
enlarged fontsize

<img width=300 src="https://github.com/user-attachments/assets/62652fdc-5fbb-4de1-9cb1-adbc40d30cd2" />